eldritch/items/unique.py

```diff
@@ -56,3 +58,228 @@ def get_trigger(self, event, owner, state):
+        rolls,
+        events.Conditional(
+            owner, rolls, "successes", {0: no_success, 1: one_success, 2: two_success}
+        )
+    ], owner)
+
+
+class BlueWatcher(Item):
+  def __init__(self, idx):
+    super().__init__("Blue Watcher", idx, "unique", {}, {}, None, 4)
+
+  def get_usable_interrupt(self, event, owner, state):
+    if not isinstance(event, (events.Check, events.CombatRound)):
+      return None
+    if not owner == event.character:
+      return None
+    if owner.stamina < 2:
+      return None
+    if isinstance(event, events.CombatRound):
+      return events.Sequence([
+          events.PassCombatRound(
+              event,
+              log_message="{char_name} passed a combat round against {monster_name} using Blue Watcher of the Pyramid"
+          ),
+          events.DiscardSpecific(owner, [self])
+      ], owner)
+    # TODO: Pass combat check or gate close check
+    if (event.check_type == "combat"
+        or (event.check_type in ("fight", "lore")
+            and isinstance(state.event_stack[-2], events.GateCloseAttempt))):
+      return events.Sequence(
+          [events.DiscardSpecific(owner, [self]), events.PassCheck(owner, event)],
+          owner
+      )
+    return None
+
+
+class EnchantedJewelry(Item):
+  def __init__(self, idx):
+    super().__init__("Enchanted Jewelry", idx, "unique", {}, {}, None, 3)
+    self.tokens["stamina"] = 0
+    self.max_tokens["stamina"] = 3
+
+  def get_usable_interrupt(self, event, owner, state):
+    if not isinstance(event, events.GainOrLoss):
+      return None
+
+    if not owner == event.character:
+      return None
+
+    loss = event.losses.get("stamina", 0)
+    if isinstance(loss, values.Value):
+      loss = loss.value(state)
+
+    if loss <= 0:
+      return None
+
+    reduction = events.LossPrevention(self, event, "stamina", 1)
+    return events.Sequence(
+        [
+            reduction,
+            events.AddToken(self, "stamina", event.character, n_tokens=1)
+        ],
+        owner
+    )
+
+
+class GateBox(Item):
+  def __init__(self, idx):
+    super().__init__("Gate Box", idx, "unique", {}, {}, None, 4)
+
+  def get_interrupt(self, event, owner, state):
+    if not isinstance(event, events.GateChoice) or not event.character == owner:
+      return None
+    if event.gate_name is None:
+      return None
+    return events.Sequence([
+        events.CancelEvent(event),
+        events.GateChoice(owner, "Choose any open gate", None, event.none_choice, event.annotation)
+    ],
+        owner)
+
+
+class HealingStone(Item):
+  def __init__(self, idx):
+    super().__init__("Healing Stone", idx, "unique", {}, {}, None, 8)
+
+  # Copied get_usable{_trigger,_interrupt,} from Physician ability
+  def get_usable_trigger(self, event, owner, state):
+    if not isinstance(event, events.UpkeepActions):
+      return None
+    return self.get_usable(event, owner, state)
+
+  def get_usable_interrupt(self, event, owner, state):
```

Comment:
I may change these to only be usable triggers on UpkeepActions. Otherwise, it can prompt the user too many times.